newrelic_lambda_cli/templates/import-template.yaml

```diff
@@ -58,8 +61,8 @@ Resources:
     DeletionPolicy: Delete
     Properties:
       CodeUri:
-        Bucket: awsserverlessrepo-changesets-1iiv3xa62ln3m
-        Key: 466768951184/arn:aws:serverlessrepo:us-east-1:463657938898:applications-NewRelic-log-ingestion-versions-2.2.1/2dcb4087-186a-42c8-bc1f-cf93780786c0
```

Comment:
@MattWhelan Doesn't this need to remain this version to ensure a clean upgrade path?

Reply:
old bucket was us-east-1, this one is in eu-west-1. Lambda requires bucket to be in same region :disappointed: 

get_sar_template_url() handles it automatically

Reply:
I removed this commit from the PR.